src/System.Net.Security/src/System.Net.Security.csproj

```diff
@@ -31,29 +31,34 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'net46_Release|AnyCPU'" />
```

Comment:
I'm confused by this as well. This probably needs to be updated to netstandard1.7 anyway? (I see #11265 doing some of those changes.)